### PR TITLE
Add go to per-seat guide and write integration test

### PR DESCRIPTION
**Pull request by cjavilla-stripe**
r? @dawn-stripe

### CODE REVIEW

File: spec/per_seat_server_spec.rb [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+
+    # Create the subscription
+    subscription, _ = post_json("/create-subscription", {
+      customerId: customer_id,
+      paymentMethodId: "pm_card_visa",
+      priceId: "BASIC",
+      quantity: 3,
+    })
+    expect(subscription).not_to be_nil
+    expect(subscription["id"]).to start_with("sub_")
```
Comment (dawn-stripe) [inline spec/per_seat_server_spec.rb:26]: should we check that the quantity was set correctly?

File: spec/per_seat_server_spec.rb [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+    # Update subscription with new price
+    updated_subscription, _ = post_json("/update-subscription", {
+      customerId: customer_id,
+      newPriceId: "PREMIUM",
+      quantity: 5,
+      subscriptionId: subscription_id
+    })
+    expect(updated_subscription).not_to be_nil
+    expect(updated_subscription.keys).to contain_exactly("subscription")
+
```
Comment (dawn-stripe) [inline spec/per_seat_server_spec.rb:137]: confirm quantity was updated in the call?

**Comment by dawn-stripe** [comment]
love the tests, thank you for kicking that off and for doing the Go work.

File: spec/per_seat_server_spec.rb (hunk 1 above)
Comment (cjavilla-stripe) [reply spec/per_seat_server_spec.rb:26]: great call! added

File: spec/per_seat_server_spec.rb (hunk 2 above)
Comment (cjavilla-stripe) [reply spec/per_seat_server_spec.rb:141]: good one. added above when the update should succeed with the quantity change.

**Comment by dawn-stripe** [comment]
lgtm @cjavilla-stripe

**Pull request merged**